packages/react-sdk/src/useConnected.ts

```diff
@@ -0,0 +1,7 @@
+import { useRiverSync } from './internals/useRiverSync'
+
+export const useConnection = () => {
+    const river = useRiverSync()
+    // TODO: check the case that there's a sync agent but it's disconnected by any reason
+    return { isConnected: !!river?.syncAgent }
+}
```

Comment:
can we delete this for now in favor of `isConnected` on useConnectRiver - you could call it useRiverConnection if you want